bindings/python/iree/runtime/system_api.py

```diff
@@ -109,7 +116,7 @@ class Config:
   device: _binding.HalDevice
   vm_instance: _binding.VmInstance
   host_type_factory: _binding.HostTypeFactory
-  default_modules: Tuple[AnyModule]
+  default_vm_modules: Tuple[_binding.VmModule, ...]
```

Comment:
Oh, thanks - I just ran into it and changed it to `Sequence`. But your way is better.